common/ideas/00_basic_ideas.txt

```diff
@@ -611,29 +612,32 @@ navigation_ideas = {
 	}
 
 	merchant_traditions = {
-		naval_tradition_from_trade = 1.0
+		ship_durability = 0.15
 		merchants = 1
 	}
 	merchant_marine = {
-		global_ship_trade_power = 0.2
+		global_ship_trade_power = 0.25
+		trade_efficiency = 0.1
 	}
 	ships_penny = {
 		global_ship_cost = -0.1
 		naval_maintenance_modifier = -0.1
 	}
+	naval_swarm = {
+		galley_power = 0.25
+		sunk_ship_morale_hit_recieved = -0.8
```

Comment:
Perhaps 80% is a bit much, considering if stacked with naval you actually get over 100% without having to do any extra work. 50% is maybe a bit more reasonable